packages/grid/_modules_/grid/hooks/features/virtualization/useGridVirtualScroller.tsx

```diff
@@ -296,6 +297,12 @@ export const useGridVirtualScroller = (props: UseGridVirtualScrollerProps) => {
         isSelected = true;
       }
 
+      const indexes: GridRowIndexes = {
+        visibleRows: currentPage.range.firstRowIndex + nextRenderContext.firstRowIndex! + i,
+        pageRows: nextRenderContext.firstColumnIndex + i,
```

Comment:
Column?

Reply:
The comment says "Index of the row in the current page." but it uses `nextRenderContext.firstColumnIndex`.

Reply:
That's a typo, fixed